File: contests/arc117/arc117_a.py
a, b = map(int, input().split())
c = list(range(-a, 0))
d = list(range(1, b + 1))
d[-1] -= sum(c) + sum(d)
assert sum(c + d) == 0
assert sum([1 for i in c if i < 0]) == b
assert sum([1 for i in d if i > 0]) == a
print(*(c + d))

File: rust/others/src/bin/arc117_a.rs
fn main() {
    proconio::input! {(mut a,mut b):(i32,i32)}
    let mut ans = Vec::new();
    let mut s = 1;
    if a < b {
        std::mem::swap(&mut a, &mut b);
        s = -1;
    }
    (1..=a).for_each(|i| ans.push((i * s).to_string()));
    (1..b).for_each(|i| ans.push((-i * s).to_string()));
    ans.push((-s * (b..=a).sum::<i32>()).to_string());
    println!("{}", ans.join(" "));
}
